itou/approvals/admin.py

```diff
@@ -112,30 +112,40 @@ class SuspensionInline(ItouTabularInline):
     extra = 0
     show_change_link = True
     can_delete = False
-    fields = ("start_at", "end_at", "reason", "created_by", "siae")
+    fields = ("start_at", "end_at", "duration", "reason", "created_by", "siae")
     raw_id_fields = ("approval", "siae", "created_by", "updated_by")
+    readonly_fields = ("duration",)
 
     def has_change_permission(self, request, obj=None):
         return False
 
     def has_add_permission(self, request, obj=None):
         return False
 
+    @admin.display(description="Durée")
+    def duration(self, obj):
+        return f"{obj.duration.days} jours"
```

Comment:
```suggestion
        return f"{obj.duration.days} jour{pluralizefr(obj.duration.days}"
```
? :angel: 
Et pareil pour les autres ?